health/egov-persister/attendance-service-persister.yml

```
@@ -0,0 +1,406 @@
+ 
+ - version: 1.0
+   name: Attendance Log
+   description: Update attendance logs and corresponding documents
+   fromTopic: update-attendance-log-health
+   isTransaction: true
+   isAuditEnabled: true
+   module: ATND
+   objecIdJsonPath: $.id
+   tenantIdJsonPath: $.tenantId
+   transactionCodeJsonPath: $.registerId
+   auditAttributeBasePath:  $.attendance.*
```

yamllint (1.35.1)

[warning] too many spaces after colon

(colons)